### PR TITLE
Enable focus shifting between text fields in login page

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js [hunk 1]
```diff
@@ -31,8 +31,38 @@ function post(path, params, method) {
...
+                    var password = utf8_to_b64($("input#password").val());
+                    post($("#login").attr("action"), {"username": username, password: password}, "POST");
+                }
+            }
+        });
+    });
+}
 
 $( document ).ready(function() {
+
```
Comment (harshanL) [inline modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js]: Check formatting of if blocks.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js]: These formatting issues has been fixed in a recent PR

File: modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js [hunk 2]
```diff
@@ -31,8 +31,38 @@ function post(path, params, method) {
...
+    $("input#username").focus();
+    $("input#username").bind('keydown',function (e) {
+        if(e.which == 10 || e.which == 13){
```
Comment (prabathabey) [inline modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js]: Fix formatting issues.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js]: These formatting issues has been fixed in a recent PR

**Pull request merged**